README.md

```diff
@@ -18,9 +18,50 @@ The unleash defined strategies are included, to support custom strategies
 use the `ClientBuilder` and call the `strategy` method to register your custom
 strategy memoization function.
 
-The crate documentation should be consulted for more detail.
+The [crate documentation](https://docs.rs/unleash-api-client/0.6.0/unleash_api_client) should be consulted for more detail.
+
+### Configuration
+The easiest way to get started with the `Client` is using the `ClientBuilder`. A simple example is provided:
+
+```rust
+let config = EnvironmentConfig::from_env()?;
+let client = client::ClientBuilder::default()
+    .interval(500)
+    .into_client::<UserFeatures>(
+        &config.api_url,
+        &config.app_name,
+        &config.instance_id,
+        config.secret,
```

Comment:
Sadly that won't work here - the method signature where we're passing this data requires ownership of a Option<String> here. It does kinda stand out a little in the docs but I think (feel free to disagree here) that most folks will have some Rust tooling setup when they're playing with the examples and should be able to see the types easily. I'm open any changes you might see here that would make this more obvious/better?

Reply:
https://github.com/Unleash/unleash-client-rust/blob/main/src/http.rs#L21 is where the underlying divergence tracks down to. We could probably make config.secret be pass by reference, though it would be an API break

Reply:
Ah, got it! I don't think it's an issue at all; sorry if it came across that way 🙈  It was mostly out of curiosity because everything else was being passed by reference.